setup.py

```diff
@@ -0,0 +1,40 @@
+import os.path as op
+
+from setuptools import setup, find_packages
+
+
+# get the version (don't import mne here, so dependencies are not needed)
+version = None
+with open(op.join('pyriemann_qiskit', '_version.py'), 'r') as fid:
+    for line in (line.strip() for line in fid):
+        if line.startswith('__version__'):
+            version = line.split('=')[1].strip().strip('\'')
+            break
+if version is None:
+    raise RuntimeError('Could not determine version')
+
+with open('README.md', 'r', encoding="utf8") as fid:
+    long_description = fid.read()
+
+setup(name='pyriemann-qiskit',
+      version=version,
+      description='Qiskit wrapper for pyRiemann',
+      url='https://pyriemann.readthedocs.io',
+      author='Alexandre Barachant',
+      author_email='alexandre.barachant@gmail.com',
```

Comment:
Must be updated.

Reply:
Sure. Can you please tell me how to deploy the docs once the artifact is generated?

Reply:
@sylvchev , can you test the documentation deployment?